### PR TITLE
fix: remove top margin on first content element

### LINKED ISSUE #776: Any first content element should have *no* top margin
The Designs assumed that every page has a subtitle and the implementation was approved on that assumption too. 

But some pages do not have a subtitle and directly start with a heading, image etc.   

Esp. in the case of a heading the heading's extra top space occurs at the top of the page, which is not intended (content should start aligned with the index navigation)

So the first element in the first section of the page content needs to have no margin top.  (will be a fancy CSS expression and we may run into stupid specificity issues)

### PR DESCRIPTION
**Pull request by TimonRey**
closes #776

### CODE REVIEW

**Review by emmenko — APPROVED** [review]
Selector looks good.

I think @Shecki or @nkuehn should check the Percy snapshots and approve them if the outcome is ok.

**Comment by Shecki** [comment]
from my side its fine, but i can´t approv

File: packages/ui-kit/src/components/markdown.js [hunk 1]
```diff
@@ -245,6 +245,10 @@ const TypographyPage = styled.div`
     margin-top: ${dimensions.spacings.m};
   }
 
+  section:first-of-type > * {
```
Comment (nkuehn) [inline packages/ui-kit/src/components/markdown.js:248]: I think the selector is matching too much.  The requirement is to only match the very first content element on a given page, not any direct descendant of the first section. 

```suggestion
  section:first-of-type > *:first-child {
```

**Review by nkuehn — CHANGES_REQUESTED** [review]
Not as intended yet, you can see in the percy shots that it#s also removing a lot of the spacing on the body content.

**Review by nkuehn — APPROVED** [review]
Thanks! good to go for this step and the correct way to do it

Only one page - the beta page - has a visual regression that makes obvious that we have no explicit space any more now (in fact, never had).  This situation will be addressed with implementing the extra space for the search bar, which will give all pages a fixed extra spacing between header and body content area independently of the specific header or body content.  

image

**Review by emmenko — APPROVED** [review]

**Pull request merged**